lib/improver/spotdata/common_functions.py

```diff
@@ -35,6 +35,8 @@
 
 import warnings
 import numpy as np
+from datetime import datetime as dt
+import iris
```

Comment:
There are still two `import iris` statements at the top of this file. Only one is needed.